htlcswitch/switch.go

```diff
@@ -1712,6 +1744,8 @@ func (s *Switch) Stop() error {
 	// accessed and modified.
 	s.mailOrchestrator.Stop()
 
+	s.blockEpochs.Cancel()
```

Comment:
This should be placed in the main goroutine as a defer instead. As otherwise, it's possible for the switch to exit, but the `Stop` method to not have been called.

Reply:
Fixed.